spec/link/group-subgroup.js

```diff
@@ -3,7 +3,8 @@ module.exports = {
   tangle: 'link',
   staticProps: {
     parent: { $ref: '#/definitions/cloakedMessageId', required: true },
-    child: { $ref: '#/definitions/cloakedMessageId', required: true }
+    child: { $ref: '#/definitions/cloakedMessageId', required: true },
+    admin: { type: 'boolean' }
```

Comment:
:+1:

Reply:
oh wait.... on second though I think this should be in `props`.
We might want to allow people to mutate this.

(the only reason parent and child and static is so links can't be moved around, as this would mess with people reviewing them )

Reply:
:fire: 

what do you think?

Reply:
Yeah i put it in static because i thought it wasnt going to be mutable, but i can move it :)